src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -68,11 +66,14 @@ export const PublishingAccordion = ({
   const customer = useSelector((store: RootState) => store.customer);
 
   const [isLoading, setIsLoading] = useState(LoadingState.None);
-  const [registrationIsValid, setRegistrationIsValid] = useState(false);
   const registrationHasFile = registration.associatedArtifacts.some((artifact) => artifact.type === 'PublishedFile');
   const completedTickets = publishingRequestTickets.filter((ticket) => ticket.status === 'Completed');
   const userCanPublish = userCanPublishRegistration(registration);
 
+  const formErrors = validateRegistrationForm(registration);
+  const registrationIsValid = Object.keys(formErrors).length <= 0;
```

Comment:
Lengden kan vel aldri være mindre enn 0, men har jo ingen praktisk betydning her da
```suggestion
  const registrationIsValid = Object.keys(formErrors).length === 0;
```